catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/dialog/ReplaceExistingProjectDialogFragment.java

```diff
@@ -122,20 +117,30 @@ public void onPositiveButtonClick(DialogInterface dialog, String textInput) {
 				.setNegativeButton(R.string.notification_download_project_cancel, null)
 				.create();
 
-		radioGroup.setOnCheckedChangeListener(new RadioGroup.OnCheckedChangeListener() {
-			@Override
-			public void onCheckedChanged(RadioGroup group, @IdRes int checkedId) {
-				switch (checkedId) {
-					case R.id.replace:
-						inputLayout.setVisibility(TextView.GONE);
-						alertDialog.getButton(AlertDialog.BUTTON_POSITIVE).setEnabled(true);
-						break;
```

Comment:
I know that this does not concern this ticket, however there seems to be an issue with the keyboard popping up when you change the selection of the radio button. Please add following code here:
`ViewUtils.hideKeyboard(inputLayout.getEditText());`